guides/Programming_Paradigm/parallel_computating/parallel_optimization.md

```diff
@@ -0,0 +1,26 @@
+# Overview 
+In all areas of engineering, optimization is a way of im proving designs,  it has served as a tool to save material, time, effort, energy, etc. When working in a new design, me chanical engineers need to analyze and test each candidate configuration, usually, in a test-error cycle they improve an initial design with respect to to a given criterion. To test a specific configuration of a mechanism, a computer simula tion is convenient in order to save resources, and reducing costs and time. In addition, an adequate computer program could test much more candidate configurations than a man designer, nevertheless, it must do it in an adequate an efficient way.
+# Simulated annealing
+The Simulated Annealing Algorithm (SA)[1] is a global optimization algorithm based on Gibbs probability distri bution. It generates random configurations from the Gibbs distribution using the metropolis sampling method, the better a configuration is the greater the probability to be sampled. Nevertheless, a drawback of the SA is the need of a huge amount of objective function evaluations, in this case an evaluation is a numerical simulation of a particular robot configuration.  The SA requires lots of simulations under different conditions to know when a configuration is better than another. Due to this fact, parallel computation is used to reduce the time require to find an approximation to the global optimal mechanism.  So, SA returns the best con figuration of a mechanism, selected among a huge amount of randomly generated configurations, having in mind that this random process samples intensively the most promising regions of the search space.
```

Comment:
The purpose of this project is to create guides that are easy to understand for rookies. Please decrease the technical jargon in this article or clarify the jargon. The other article seems good to go